test/ui-test/testSuites/suite_status/tst_statusSignUp/test.feature

```diff
@@ -11,65 +11,191 @@
+
+        # Input wrong password in final password input and verify password creation button is not enabled
+        When user inputs the following <wrongpassword> with ui-component loginView_passwordInput
+        Then the following ui-component mainWindow_Finalise_Status_Password_Creation_StatusBaseText is not enabled
+
+        Examples:
+            | username  | wrongpassword        | password         |
+            | tester123 | Invalid34            | TesTEr16843/!@00 |
+            | tester123 | TesTEr16843/!@)      | TesTEr16843/!@01 |
+
+
+        Scenario: User signs up with imported 12 seed phrase
+
+        Given A first time user lands on the status desktop and navigates to import seed phrase
+        When The user inputs 12 seed phrases
+            | phrases   | occurrence |
+            | lawn      | 1   	     |
+            | corn      | 3   	     |
+            | paddle    | 5          |
+            | survey    | 7          |
+            | shrimp    | 9          |
+            | mind      | 11         |
+            | select    | 2    	     |
+            | gaze      | 4          |
+            | arrest    | 6          |
+            | pear      | 8          |
+            | reduce    | 10         |
+            | scan      | 12         |
+        And user clicks on the following ui-component mainWindow_submitButton_StatusButton
+        When user signs up with username tester123 and password TesTEr16843/!@00
+        Then the user lands on the signed in app
+        When the user opens app settings screen
+        And  the user activates wallet and opens the wallets section
+        Then the 12 seed phrase address is 0x8285cb9bf17b23d64a489a8dad29163dd227d0fd displayed in the wallet
+
+
+    Scenario: User signs up with imported 18 seed phrase
+
+        Given A first time user lands on the status desktop and navigates to import seed phrase
+        When The user inputs 18 seed phrases
+            | phrases   | occurrence |
+            | flip      | 1   	     |
+            | foam      | 4          |
+            | time      | 7    	     |
+            | sight     | 10         |
+            | scheme    | 13    	 |
+            | describe  | 16         |
+            | candy     | 2   	     |
+            | erosion   | 5          |
+            | layer     | 8          |
+            | depth     | 11         |
+            | extend    | 14         |
+            | dish      | 17         |
+            | fog       | 3          |
+            | seven     | 6          |
+            | budget    | 9          |
+            | denial    | 12         |
+            | body      | 15         |
+            | device    | 18         |
+
+        And user clicks on the following ui-component mainWindow_submitButton_StatusButton
+        When user signs up with username tester124 and password TesTEr16843/!@00
+        Then the user lands on the signed in app
+        When the user opens app settings screen
```

Comment:
Not sure why opening settings and wallet section is part of sign up suite

Reply:
Sorry, I don't have access to `TestRail`.. so, is it part of `SignUp` test case?